Added logging

tomekczm opened the pull request:
Logs british people, message deletion, message edition, member updates, role updates, guild channel updates, joins leaves and more i think. I also wanted to do channel permissions updates but its too hard and probably useless

Commit:
Added welcome messages

mat-1 committed:
Improve welcome embed

mat-1 committed:
replace some spaces with tabs

Commit:
Added basic logging

tomekczpog commented:
![image](https://user-images.githubusercontent.com/74684114/156927080-28edee73-4de9-4367-bc88-83b54b1797d3.png)

tomekczm committed:
my repo was outdated :GB:

tomekczm committed:
Merge branch 'master' into master

Commit:
Comment out the filter

Commit:
Fixed the bot logging its own messages being edited or deleted

Commit:
Fix bot moderating its own messages + fixed empty logs (hopefully)

tomekczm committed:
fixed welcome and added run.sh for utility

tomekczm committed:
more filters removed

tomekczm committed:
Update everything

tomekczm committed:
Update tetris.py

tomekczm committed:
stuff

tomekczm committed:
hopefully fix stuff

tomekczm committed:
Big changes (HOPE NOTHING BREAKS)

tomekczm committed:
forgor a await

tomekczm committed:
trollface

tomekczm committed:
im dum

tomekczm committed:
Ticket improvements & more

and quick prompt

tomekczm committed:
Added delay & more

also fixed empty logs (hopefully) added improvements and cooldown for tickets!!!

tomekczm committed:
pissylegs reviews

tomekczm committed:
dasdasd

tomekczm committed:
im dumb

tomekczm committed:
like piglegs messages

tomekczm committed:
Added files argument to send_log_message

tomekczm committed:
add place account creation

tomekczm committed:
added a thing to quickly switch between production and dev configs

tomekczm committed:
ban monday from saying *

tomekczm committed:
Update modbot.py

tomekczm committed:
Update modbot.py

troll

tomekczm committed:
Major bug fixes

morocco fixed
